conda_forge_tick/url_transforms.py

```diff
@@ -48,23 +50,38 @@ def _v_munger(url):
 
 
 def _pypi_domain_munger(url):
-    for old_d, new_d in permutations(
-        ["https://pypi.io", "https://files.pythonhosted.org"],
-        2,
-    ):
+    for old_d, new_d in permutations(PYPI_URLS, 2):
         yield url.replace(old_d, new_d, 1)
 
 
 def _pypi_name_munger(url):
     bn = os.path.basename(url)
-    if (
-        url.startswith("https://pypi.io")
-        or url.startswith("https://files.pythonhosted.org")
-    ) and ("{{ version }}" in bn and "{{ name" not in bn):
-        yield os.path.join(os.path.dirname(url), "{{ name }}-{{ version }}.tar.gz")
```

Comment:
It looks like you removed this variant that we try. We should yield it too.

Reply:
Hm... it doesn't check to see if that variable is even set... seems like _adding_ variables when not asked could potentially make things _worse_...

Reply:
If the url doesn't work it tries the next one, so it doesn't matter. Also there is a test if name is in the url in the first place. That means it is likely there in the recipe. 

In general though, please do not remove features. I've seen this more than once in PRs others have made to the bot and I don't really understand it. 

Everything is here for a reason, even if we do not recall why. Unless we know what happened, we should leave things in place. 

Let's add this back and then we can merge.

Reply:
2bea2c2f